src/applications/claims-status/components/DueDate.jsx

```diff
@@ -2,17 +2,21 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import moment from 'moment';
 
+import { DATE_FORMATS } from '../constants';
+import { buildDateFormatter } from '../utils/helpers';
+
 export default function DueDate({ date }) {
   const now = moment();
   const dueDate = moment(date);
   const className = dueDate.isBefore(now) ? 'past-due' : 'due-file';
+
+  const formatDate = buildDateFormatter(DATE_FORMATS.LONG_DATE);
```

Comment:
This might work, but I would be careful with mixing moment and date-fns

Reply:
This is what the component previously had (now called DueDateOld.jsx) so I just kept it! Happy to change the logic here to something else if theres a way we can improve it! @jerekshoe